experiments/full_fov_15km.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+"""
+Copyright SuperDARN Canada 2022
+The mode transmits with a pre-calculated phase progression across the array which illuminates the full FOV,
+and receives on all antennas. The first pulse in each sequence starts on the 0.1 second boundaries, to enable bistatic
+listening on other radars. This mode uses 15-km range gates for high spatial resolution.
+"""
+
+import sys
+import os
+
+BOREALISPATH = os.environ['BOREALISPATH']
+sys.path.append(BOREALISPATH)
+
+import experiments.superdarn_common_fields as scf
+from experiment_prototype.experiment_prototype import ExperimentPrototype
+from experiment_prototype.decimation_scheme.decimation_scheme import \
+    DecimationScheme, DecimationStage, create_firwin_filter_by_attenuation
+
+
+def create_15km_scheme():
```

Comment:
Ah,  another opportunity to refactor. This time -> Decimation scheme code. If we're happy with the 15km decimation scheme as we ran it in IB_Collab_mode, then we should roll that into the default option in decimation_scheme.py

Reply:
I haven't a clue how that the experiment went whenever they ran it, but I could take a look at the filter steps and see if they seem reasonable